src/components/ButtonWithDropdownMenu.js

```diff
@@ -87,9 +95,7 @@ const ButtonWithDropdownMenu = (props) => {
                         success
                         isDisabled={props.isDisabled}
                         style={[styles.pl0]}
-                        onPress={() => {
-                            setIsMenuVisible(true);
-                        }}
+                        onPress={measurePopoverPositionAndShowMenu}
```

Comment:
```suggestion
                        onPress={() => measurePopoverPositionAndShowMenu()}
```
onPress will pass an event to the function